### PR TITLE
Added query log admonition

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Document new query log filtering limitations/restrictions](https://trello.com/c/zlPd8BBe/282-document-new-query-log-filtering-limitations-restrictions)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-314.surge.sh

Updated pages:
- [Logs](https://neo4j-docs-aura-314.surge.sh/aura/platform/logging)

File: modules/ROOT/pages/platform/logging.adoc [hunk 1]
```diff
@@ -13,7 +13,12 @@ To access the *Logs* tab:
 
 == Query logs
 
-A query log provides a log of queries executed on an instance within a specified time range. 
+A query log provides a log of queries executed on an instance within a specified time range.
+
+[NOTE]
+====
+Queries that complete succesfully within 50ms are not logged.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/logging.adoc:20]:
```suggestion
Queries that complete successfully within 50ms are not logged.
```

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/logging.adoc:20]: how grammarly doesn't see this as a spelling mistake I have no fucking idea.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-314.surge.sh

Updated pages:
- [Logs](https://neo4j-docs-aura-314.surge.sh/aura/platform/logging)

**Review by nvitucci — APPROVED** [review]

**Pull request merged**